### ISSUE TITLE
Pihole Query log not showed

### ISSUE THREAD

**Issue by dlevequeIT** [pi-hole/pi-hole#4779]
### Versions
Pi-hole version is v5.10 (Latest: v5.10)
  AdminLTE version is v5.12-6-g61b02cbd (Latest: v5.12)
  FTL version is v5.15 (Latest: v5.15)

### Platform
- OS and version:  Ubuntu 18.04
- Platform:  Server custom with apache, bind, php-8.1

### Expected behavior

On the Web interface the 'Query log' page stay empty instead of the 'Dashboard' page show the counts increase.
It's the same in the 'Group', 'Whitelist', 'Blacklist' sections.

When I run 'pihole debug' I have this: https://tricorder.pi-hole.net/gUCnCGmG/
I think that my issue is due to this element:

> [ DIAGNOSING ]: Dashboard and block page
[✗] Block page X-Header: X-Header does not match or could not be retrieved.
HTTP/1.1 404 Not Found
Date: Mon, 04 Jul 2022 17:09:47 GMT
Server: Apache
Content-Type: text/html; charset=iso-8859-1

> [✗] Web interface X-Header: X-Header does not match or could not be retrieved.
HTTP/1.1 404 Not Found
Date: Mon, 04 Jul 2022 17:09:47 GMT
Server: Apache
Content-Type: text/html; charset=iso-8859-1

But I don't know how to resolv that

**Comment by dft601** [comment]
Yes have the same problem... My Android app can also not show the query log since 5.10.
There are some more user reporting this.
https://play.google.com/store/apps/details?id=com.tien.piholeconnect&hl=de&gl=US

**Comment by rdwebdesign** [comment]
Wait... I doubt you are talking about the same problem.

@dft601
You are talking about a third party app. Please, report your bug for the app developers. 

@dlevequeIT
The errors you posted are here because you are using `apache` as your web server. Pi-hole uses `lighttpd`.
I read you debug log (quickly) and looks like you have a lot of customization.
We need more time to find the real problem.

**Comment by jfb-pihole** [comment]
> Yes have the same problem... My Android app can also not show the query log since 5.10. There are some more user reporting this. https://play.google.com/store/apps/details?id=com.tien.piholeconnect&hl=de&gl=US

You have a different problem, and the app you are showing in your link is not an app provided or supported by the Pi-hole team.  For support with that app, contact the app developer.

**Comment by jfb-pihole** [comment]
> But I don't know how to resolv that

Either uninstall Apache so that lighttpd can bind to port 80, or move one of the two webservers off of port 80, so they can both run.

To move lighttpd to an alternate port, see this thread:

https://discourse.pi-hole.net/t/lighttpd-daemon-wont-start/44985/9

**Comment by dft601** [comment]
@jfb-pihole   I talked to the developer of the app,  He told me:
This issue was actually a problem with pi hole and was already fixed by their developers: https://github.com/pi-hole/AdminLTE/issues/2178

The fix just hasn’t been released yet

---
Regards,
Tien

**Comment by dft601** [comment]
okay for me with new release

**Comment by dlevequeIT** [comment]
I tried to reinstall from the GIT repository, restart my server, and now it works.
Only my android Apps (with the API key usage) refuses to display realtime log activity

**Comment by vandabbin** [comment]
I can view the query log in a browser on my laptop, chrome or firefox, however on an android mobile device, the query log is only viewable on firefox and not in chrome.
